app/src/org/commcare/android/database/user/models/FormRecordV1.java

```diff
@@ -0,0 +1,78 @@
+package org.commcare.android.database.user.models;
+
+import org.commcare.android.storage.framework.Persisted;
+import org.commcare.android.storage.framework.Persisting;
+import org.commcare.android.storage.framework.Table;
+import org.commcare.modern.models.EncryptedModel;
+import org.commcare.modern.models.MetaField;
+
+import java.util.Date;
+
+/**
+ * This class represents the version of a FormRecord that exists on any devices running a pre-2.26
+ * version of CommCare, which was deprecated in user db version 9. This class is used to read a
+ * FormRecord that exists in such a database, in order to run a db upgrade.
+ *
+ * @author amstone
+ */
+@Table(FormRecord.STORAGE_KEY)
```

Comment:
For the legacy records it's good to manually code this in, since the value of static fields can change over time, and the values and structure of fixed models don't.